### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExplorer_ScreenReader: VoiceOver does not read the necessary headings on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52443]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Explorer"  button.
3. Navigate to "Explorer" section and read the content "Explorer" using basic navigation. Read headings using Ctrl + Option + Command + H.

**Actual:**
Screenreader does not read the content "Explorer" as appropriate heading which is so.

**Expected:**
Screenreader should read- "Explorer" as heading with appropriate heading level.

**Recommendations:**
Add appropriate native heading level tag to the content "Explorer".
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the appropriate headings present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_HeadingsIssue.pptx](https://github.com/Microsoft/vscode/files/2118875/A11y_VSCode_HeadingsIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51760
